### PR TITLE
 Free some memory in unit tests

### PR DESCRIPTION
**Pull request by bancer**

### CODE REVIEW

**Comment by bancer** [comment]
SQLite test failed to load PHPUnit. Please restart the build.

File: lib/Cake/TestSuite/CakeTestCase.php [hunk 1]
```diff
@@ -164,6 +166,8 @@ public function tearDown() {
 		if (isset($_GET['debug']) && $_GET['debug']) {
 			ob_flush();
 		}
+		unset($this->_configure);
+		unset($this->_pathRestore);
```
Comment (markstory) [inline lib/Cake/TestSuite/CakeTestCase.php]: You can do all of this in one unset call like `unset($this->_configure, $this->_pathRestore)`.

File: lib/Cake/TestSuite/ControllerTestCase.php [hunk 2]
```diff
@@ -418,4 +418,18 @@ public function generate($controller, $mocks = array()) {
...
+ * @return void
+ */
+	public function tearDown() {
+		parent::tearDown();
+		unset($this->contents);
+		unset($this->controller);
+		unset($this->headers);
+		unset($this->result);
+		unset($this->view);
+		unset($this->vars);
```
Comment (markstory) [inline lib/Cake/TestSuite/ControllerTestCase.php]: You could combine all of this into one unset() as well.

Comment (bancer) [reply lib/Cake/TestSuite/ControllerTestCase.php]: Of course if that is a preferable syntax. One line or multiple?

**Pull request merged**